specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -2629,6 +2629,100 @@
           }
         }
       }
+    },
+	"/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/customers/{customerName}/policies/default": {
```

Comment:
Remove /billingProfiles/{billingProfileName}. The "home" resource container for customers is /providers/Microsoft.Billing/billingAccounts/{billingAccountName}/customers.

Reply:
done